gatsby-config.js

```diff
@@ -2,6 +2,7 @@ require(`dotenv`).config()
 
 const siteUrl = process.env.SITE_URL || `https://covid.citybureau.org`
 
+// languages to support
```

Comment:
I think you clarified this in your Readme, but available languages are listed in several places and I was curious if the list has to be modified in more than one place, or if just taking languages off the list here would effectively limit a fork of the site to supporting only a given array of languages.